03week/towersOfHanoi.js

```diff
@@ -19,24 +19,53 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
-
+//make the 2 tests for this function 
+const movePiece = (valueInArrB4Move, valueInArrB4Drop) => {
+  if(isLegal(valueInArrB4Move[valueInArrB4Move.length-1], valueInArrB4Drop[valueInArrB4Drop.length-1])){
```

Comment:
movePiece shouldn't be calling isLegal, this is spaghetti code and is difficult to maintain.